scripts/setup_db.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
```

Comment:
this looks good
can you convert `scripts/setup_db.sh` to `xtask`?

it basically means something like:

```
$ cargo xtask setup_db
```

Also see if you can print a notice that "A postgres database has been started in the background"

In addition moving to `xtask` means you write pure rust, so maybe you can do something like

```
$ cargo xtask setup_db --user loco --pass loco --db loco_app
```

and if no flags given, use default values

Reply:
xtask code starts here: https://github.com/loco-rs/loco/blob/master/xtask/src/bin/main.rs

Reply:
BTW usage of default PG port isn't recommended in my point of view.
I have PG installed and I always have issues with it since the docker and my local install are supposed to use the same port.

Using non-default ensures it's a smoother ride for any user, regardless if they have PG locally or not.

Reply:
Removed, seems like a local issue in my env.